scripts/rebuild-fixtures.ts

```diff
@@ -0,0 +1,20 @@
+import fs from 'node:fs/promises';
+import path from 'node:path';
+import { jsonBinaryBlobCodec } from '../test/json-binary-blob-codec.js';
+
+const { replacer } = jsonBinaryBlobCodec;
+
+const pathname = new URL(import.meta.url).pathname;
+const __dirname = path.dirname(pathname);
+
+const files = await fs.readdir(path.resolve(__dirname, '../test/fixture'));
```

Comment:
Does [`import.meta.resolve()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import.meta/resolve) work for this? 

```suggestion
const files = await fs.readdir(import.meta.resolve('../test/fixture'));
```

Reply:
tried but path.dirname really doesn't like the file URL for some reason

Reply:
Would [`import.meta.dirname`](https://nodejs.org/docs/latest-v22.x/api/esm.html#importmetadirname) work?

Reply:
ah yeah that'd work for the `const __dirname` piece